adb_graphics/utils.py

```diff
@@ -34,6 +34,7 @@ def fhr_list(args):
     args = args if isinstance(args, list) else [args]
     arg_len = len(args)
     if arg_len in (2, 3):
+        args[-1] += 1
```

Comment:
Will allow for inclusive forecast hour -- when running on command line `-f 0 6` previously would only plot 0-5 hour graphics.